sechub-developertools/scripts/sechub-api.sh

```diff
@@ -316,11 +344,6 @@ while [[ "${opt:0:1}" == "-" ]] ; do
     echo "### Connecting as $SECHUB_USERID to $SECHUB_SERVER"
     shift
     ;;
-  *)
```

Comment:
Is there a particular reason for removing this case?

Reply:
Yes.
It could happen that usage was printed twice.
The case is already covered by line 492.